views/help/forms/form_wish.hbs

```diff
@@ -2,56 +2,55 @@
+	<span class="control-label">{{$t "help.form_Wish.label.iWouldLikeTo" }}</span>
+	<textarea class="form-control" cols="50" name="desire" rows="6" placeholder="{{$t "help.form_Wish.placeholder.canEasilyFindAndChange" }}" required></textarea>
 </label>
 <label class="form-group">
-	<span class="control-label">um ...</span>
-	<textarea class="form-control" cols="50" name="benefit" rows="6" placeholder="z.B. sie zu aktualisieren" required></textarea>
+	<span class="control-label">{{$t "help.form_Wish.label.around" }}</span>
+	<textarea class="form-control" cols="50" name="benefit" rows="6" placeholder="{{$t "help.form_Wish.placeholder.toUpdateThem" }}" required></textarea>
 </label>
 <label class="form-group">
-	<span class="control-label optional">Akzeptanzkriterien</span>
-	<textarea class="form-control" cols="50" name="acceptanceCriteria" rows="6" placeholder="z.B. Nach der Systemanmeldung gelange ich in maximal zwei Schritten zum Stundenkontext, kann die Schüleraufgabe dort mit einem Klick bearbeiten und die SchülerInnen bei Bedarf automatisch über die Änderung informieren."></textarea>
+	<span class="control-label optional">{{$t "help.form_Wish.label.acceptanceCriteria" }}</span>
+	<textarea class="form-control" cols="50" name="acceptanceCriteria" rows="6" placeholder="{{$t "help.form_Wish.placeholder.afterLoggingOnToTheSystem" }}"></textarea>
 </label>
 <label class="form-group">
-	<span class="control-label optional">Gerät</span>
+	<span class="control-label optional">{{$t "help.form_Wish.label.device" }}</span>
 	<input type="text" name="device" placeholder="z.B. iPhone X, Samsung Galaxy S10" class="form-control">
 </label>
 <label class="form-group">
 	<span class="control-label" for="email">
-		E-Mail-Adresse für eventuelle Rückfragen
+		{{$t "help.form_Wish.label.replyEmail" }}
 	</span>
-	<input type="email" name="replyEmail" placeholder="E-Mail-Adresse" value="{{ userEmail }}" class="form-control" required>
+	<input type="email" name="replyEmail" placeholder={{$t "help.placeholder.email"}} value="{{ userEmail }}" class="form-control" required>
 </label>
 <hr>
-<p>Weitere Informationen (optional):</p>
+<p>{{$t "help.global.label.furtherInformation" }}</p>
 <label>
 <div class="row">
 	<div class="col-xs-2">
 		<input type="checkbox" id="consent" name="consent" value="true" checked>
 	</div>
 	<div class="col-xs-10">
 		<span class="control-label" for="consent">
-			Ich bin damit einverstanden,
-			dass zur besseren Bearbeitung meines Wunsches folgende Informationen mitgeschickt werden dürfen:
-			<li>Browser Name und Version (z.B: Firefox 69.0)</li>
-			<li>Betriebssystem und Version (z.B: Windows 10)</li>
+			{{$t "help.form_Wish.text.iAgreeThatTheFollowingInformation" }}
+			<li>{{$t "help.global.text.browserNameAndVersion" }}</li>
+			<li>{{$t "help.global.text.operatingSystemAndVersion" }}</li>
```

Comment:
```suggestion
			{{{$t "help.form_bug.text.iAgreeThatTheFollowingInformationMayBe" }}}
```
-> no duplicates and easier to translate when using one key instead of 3

Reply:
done ;)